tapdb/universe.go

```diff
@@ -252,7 +252,7 @@ func upsertAssetGen(ctx context.Context, db UpsertAssetStore,
 			Witness: genesisProof.Asset.PrevWitnesses[0].TxWitness,
 		}
 		if len(reveal.TapscriptRoot) != 0 {
-			copy(fullGroupKey.TapscriptRoot[:], reveal.TapscriptRoot)
+			copy(fullGroupKey.TapscriptRoot, reveal.TapscriptRoot)
```

Comment:
I don't think we need the `id` and the `copy` anymore, can just directly assign.